packages/components/text/src/Text.types.ts

```diff
@@ -1,20 +1,21 @@
-import { ITextTokens, IForegroundColorTokens } from '@fluentui-react-native/tokens';
-import { ITextProps as INativeTextProps } from '@fluentui-react-native/adapters';
-
-export const textName = 'RNFText';
-
-/**
- * Properties for fabric native text field, these extend the default props for text
- */
-export type ITextProps<TBase = INativeTextProps> = TBase &
-  ITextTokens &
```

Comment:
should the text tokens stay part of the props?

Reply:
I asked this in an earlier conversation on Teams. We decided to only have variant as a prop to start. This way it pushes people to customize text using variants rather than setting family/weight/size directly. We can always add those props later if we decide to do so.